src/js/components/hocs.js

```diff
@@ -105,25 +105,29 @@ export const withRestrictScroll = (WrappedComponent) => {
 export const restrictFocusTo = (WrappedComponent) => {
   class FocusedContainer extends Component {
     componentDidMount() {
-      const { restrictScroll } = this.props;
-      const child = findDOMNode(this.ref);
-      getBodyChildElements()
-        .filter(isNotAncestorOf(child))
-        .forEach(makeNodeUnfocusable);
+      const { restrictScroll, position } = this.props;
```

Comment:
This seems like it should be taking a "hidden" boolean, since FocusedContainer doesn't know anything about positioning otherwise.